### PR TITLE
Unifying link styling

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #842

### CODE REVIEW

File: scss/main.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
 @import './overrides/variables.scss';
 @import '../node_modules/bootstrap/scss/mixins';
 @import '../node_modules/mofo-bootstrap/src/scss/overrides/mixins.scss';
+@import '../node_modules/bootstrap/scss/reboot.scss';
```
Comment (mmmavis) [inline scss/main.scss]: to feed our new `$link-decoration` value into `reboot.scss`...

**Comment by mmmavis** [comment]
@kristinashu help me check if I got everything right? https://network-pulse-staging-pr-869.herokuapp.com 😉

**Comment by kristinashu** [comment]
Link styles look good but I noticed the "learn more" link is missing from /feature 

Link is on production https://www.mozillapulse.org/featured but not staging https://network-pulse-staging-pr-869.herokuapp.com/featured

**Comment by mmmavis** [comment]
@kristinashu Nice catch. The missing link was because I forgot to set an environment variable on Heroku for this particular review app. Link should be there once we merge this PR to staging/production.

Assigning this review to you now @gvn . I've got Kristina reviewed the design bits already.

**Comment by mmmavis** [comment]
🎉

**Pull request merged**